index.html

```diff
@@ -0,0 +1,136 @@
+    <!-- Top Navigation -->
+    <section class="navigation">
+      <nav class="navigation__top-nav">
+        <ul>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-menu">
+              <use xlink:href="assets/Icons/sprite.svg#icon-facebook"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-cross">
+              <use xlink:href="assets/Icons/sprite.svg#icon-twitter"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">English</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">My Page</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">Logout</a>
+          </li>
+        </ul>
+      </nav>
+    </section>
+
+    <!-- Header Section -->
+    <header class="header">
+
+      <!-- Headre Nav -->
+      <section class="header-nav">
+        <div class="header-nav-container">
+          <div class="logo-box" id="logo-box-nav"><span>Art</span>~Blog</div>
+          <nav class="header-nav-container__nav-bar">
+            <div class="header-nav-container__nav-bar--icon" id="hamberger">
+              <svg class="header-nav-container__nav-bar--icon" id="icon-menu">
+                <use xlink:href="assets/Icons/sprite.svg#icon-menu"></use>
+              </svg>
+              <svg class="header-nav-container__nav-bar--icon" id="icon-cross">
+                <use xlink:href="assets/Icons/sprite.svg#icon-cross"></use>
+              </svg>
+            </div>
+            <ul class="header-nav-container__nav-bar--link">
+              <li class="nav-link"><a href="pages/about.html">About</a></li>
+              <li class="nav-link"><a href="">Program</a></li>
+              <li class="nav-link"><a href="">Join</a></li>
+              <li class="nav-link"><a href="">Sponsor</a></li>
+              <li class="nav-link"><a href="">News</a></li>
+            </ul>
+            <button class="your-blog">Your Blog</button>
+          </nav>
+        </div>
+      </section>
+
+      <!-- Header Body -->
+      <section class="header-body">
+        <p class="header-body__greeting-text">"Hello! Share your art"</p>
+        <h1 class="header-body__header-title">
+          <span>Creativity in Action</span>
+          <span>ArtBlog 2022</span>
+        </h1>
+        <div class="header-body__header-desc">
+          <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Doloremque beatae accusamus tempora voluptatibus
+            laudantium dolor molestias suscipit, ratione tenetur, delectus harum a perspiciatis consequuntur, totam non.
+            Aperiam deleniti illo dolorem!</p>
+        </div>
+        <p class="header-body__event-time">2022.03.16(WED) ~ 18(FRI)</p>
+        <p class="header-body__event-place">@ National Museum of Korea, Art Center Nabi and More</p>
+      </section>
+    </header>
```

Comment:
- [ ] As the previous reviewer requested, please fix the spacings of the main section on the home page to match the given design. 🙏 😄